packages/@react-aria/tag/src/useTag.ts

```diff
@@ -54,30 +54,27 @@ export function useTag(props: AriaTagProps): TagAria {
     }
   }
   const pressProps = {
-    onPress: e => onRemove(children, e)
+    onPress: e => onRemove && onRemove(children, e)
```

Comment:
```suggestion
    onPress: e => onRemove?.(children, e)
```
shorter way to write this

Reply:
Additionally, it may be handier to return the tag's key in onRemove in case a tag has the same text as another tag. 

An argument can be made that people wouldn't have the same tag text in multiple tags and that v2 just returned the tag text on tag removal though but IMO it would be nice to get the key back to guarantee uniqueness.